wire-ios-data-model/Source/ManagedObjectContext/Migration/2.110-2.111/PrefillPrimaryKeyAction.swift

```diff
@@ -0,0 +1,53 @@
+////
+// Wire
+// Copyright (C) 2023 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+class PrefillPrimaryKeyAction: CoreDataAction, CoreDataAction2111 {
```

Comment:
Can we use composition over inheritance with error-prune overriding methods etc.?

Reply:
We keep the class as return type for now